code/game/dna/genes/vg_powers.dm

```diff
@@ -42,15 +42,13 @@ Obviously, requires DNA2.
 		if(!istype(M)) return
 		if(HULK in M.mutations)
 			var/timeleft=M.hulk_time - world.time
-			if(M.health <= 25 || timeleft <= 0)
+			if(M.health <= 0 || timeleft <= 0)
```

Comment:
Yeah, that's what it was, or if we wanted to enable this as a an alternative mutation for some specific purpose---I didn't want it to be "hey, Hulk doesn't work the same with this"...if that ever happens (doubt it will).